### ISSUE TITLE
Hide settings menu

### ISSUE THREAD

**Issue by kartongsaft** [MarkHedleyJones/dmenu-extended#60]
Can _-> Settings_ be hidden from the menu?

**Comment by MarkHedleyJones** [comment]
Hi, 
Removing the Settings plugin would mean that users could then lock themselves out of being able to control the menu's settings (if they didn't know where to find the configuration file).
Because of that, I'd be hesitant to adding that as a feature.
However, if you open dmenu_extended.py and modify the line (728) from:

    cache_plugins = self.cache_open(file_cache_plugins)

to

    cache_plugins = ""

then the menu won't display any plugins when opened.

Note that to see the change you'll have to run the script in place `python dmenu_extended.py` (from within the dmenu-extended directory), or install the modified package on your system again (for a global installation that's `sudo python setup.py install`).

I hope that helps.

**Comment by kartongsaft** [comment]
Yes, the _Settings_ item disappears from the menu, just the way I like it. I think this should be configurable to at least give the user the option to disable this without altering the code.
